Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -557,6 +576,71 @@ - (void)sectionEditorFinishedEditing:(SectionEditorViewController*)sectionEditor
     [self fetchArticle];
 }
 
+
+
+
+
+
+
+
+
+
+
+
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController*)previewingContext:(id<UIViewControllerPreviewing>)previewingContext
+                      viewControllerForLocation:(CGPoint)location {
+
+//return
+//[[WMFArticleContainerViewController alloc] initWithArticleTitle:[[MWKTitle alloc] initWithURL:[NSURL URLWithString:@"http://en.wikipedia.org/wiki/Cat"]]
```

Comment:
Oh never mind, I see you did this already by disabling `handleTouchEnded`. Unfortunately there's not much I can do to figure this out w/o a device. @coreyfloyd would you have time to play around with this?